### PR TITLE
overall comments yes/edit/no working, final decision working

### PR DESCRIPTION
**Pull request by Robert-Krajcik**
overall comments yes/edit/no working, final decision working
Addresses issue #168

### CODE REVIEW

**Comment by marlycormar** [comment]
Hi @Robert-Krajcik, thanks for working on this issue! 

I jus tested the PR and my only comment so far is that after submitting the final decision the corresponding ui should be disabled (idem the final comment).

**Comment by Robert-Krajcik** [comment]
Ready for you to review this again, @marlycormar

File: R/sidebar.R [hunk 1]
```diff
@@ -185,39 +185,50 @@ sidebarServer <- function(id, uploaded_pkgs) {
                No - Exits from window and removes the text in comment box."
           ),
           footer = tagList(
-            actionButton("submit_overall_comment_yes", "Yes", class = "btn-success"),
-            actionButton("submit_overall_comment_edit", "Edit", class = "btn-secondary"),
-            actionButton("submit_overall_comment_no", "No", class = "btn-unsuccess")
+            actionButton(NS(id, "submit_overall_comment_yes"), "Yes", class = "btn-success"),
+            actionButton(NS(id, "submit_overall_comment_edit"), "Edit", class = "btn-secondary"),
+            actionButton(NS(id, "submit_overall_comment_no"), "No", class = "btn-unsuccess")
```
Comment (AARON-CLARK) [inline R/sidebar.R:190]: Oh looky here! I've never actually used `NS()` on each UI id, but I see that's what Hadley is doing in Mastering Shiny. Previously, we always started any UI script with: 

`ns <- NS(id)`

and then called `ns()` in it's place, so line 188 above would start out like...

`actionButton(ns("submit_overall_comment_yes"), "Yes", class = "btn-success"),`

Going forward, do we want to use little ns or big ns?

**Review by AARON-CLARK — APPROVED** [review]
Looks great Robert! I just asked one small question regarding using little or big ns, but I'm happy to merge either way. Thanks for the PR!

File: R/sidebar.R (hunk 1 above)
Comment (Robert-Krajcik) [reply R/sidebar.R:190]: As long as we're consistent, @AARON-CLARK 
Yes, I'm OK with using little ns.

Comment (Robert-Krajcik) [reply R/sidebar.R:190]: Actually, when I do
image
I find all the other functions are using big NS()
So maybe we should just leave it be.

Comment (AARON-CLARK) [reply R/sidebar.R:190]: Let's leave it as-is then!

**Pull request merged**